### PR TITLE
Remove emphasis on WritableStream controller

### PR DESCRIPTION
**Pull request by dcecile**
### Description

Remove repeated references to write controller

### Motivation

This controller can't really be used to "control the stream". It just provides an abort `signal` and an `error` method.

### Additional details

https://streams.spec.whatwg.org/#ws-default-controller-class

### CODE REVIEW

File: files/en-us/web/api/writablestream/writablestream/index.md [hunk 1]
```diff
@@ -23,23 +23,18 @@ new WritableStream(underlyingSink, queuingStrategy)
 - `underlyingSink` {{optional_inline}}
 
   - : An object containing methods and properties that define how the constructed stream
-    instance will behave. `underlyingSink` can contain the following:
+    instance will behave.  The `controller` parameter passed to this object's methods is a {{domxref("WritableStreamDefaultController")}} that provides abort and error signalling. `underlyingSink` can contain the following:
```
Comment (wbamberg) [inline files/en-us/web/api/writablestream/writablestream/index.md:26]:
```suggestion
    instance will behave.  The `controller` parameter passed to this object's methods is a {{domxref("WritableStreamDefaultController")}} that provides abort and error signaling. `underlyingSink` can contain the following:
```

US spelling 😭 .

File: files/en-us/web/api/writablestream/writablestream/index.md [hunk 2]
```diff
@@ -23,23 +23,18 @@ new WritableStream(underlyingSink, queuingStrategy)
 - `underlyingSink` {{optional_inline}}
 
   - : An object containing methods and properties that define how the constructed stream
-    instance will behave. `underlyingSink` can contain the following:
+    instance will behave.  The `controller` parameter passed to this object's methods is a {{domxref("WritableStreamDefaultController")}} that provides abort and error signaling. `underlyingSink` can contain the following:
```
Comment (wbamberg) [inline files/en-us/web/api/writablestream/writablestream/index.md:26]:
```suggestion
    instance will behave. The `controller` parameter passed to this object's methods is a {{domxref("WritableStreamDefaultController")}} that provides abort and error signaling. `underlyingSink` can contain the following:
```

**Review by wbamberg — APPROVED** [review]
👍 yes, I think this is better. Thank you @dcecile !

**Comment by dcecile** [comment]
Great, thanks @wbamberg!

**Pull request merged**